cmd/zc_traverser_s3_service.go

```diff
@@ -104,17 +106,38 @@ func (t *s3ServiceTraverser) traverse(preprocessor objectMorpher, processor obje
 		err = bucketTraverser.traverse(preprocessorForThisChild, processor, filters)
 
 		if err != nil {
+			var errStr string
 			if strings.Contains(err.Error(), "301 response missing Location header") {
-				LogStdoutAndJobLog(fmt.Sprintf("skip enumerating the bucket %q , as it's not in the region specified by source URL", v))
-				continue
+				errStr = fmt.Sprintf("skip enumerating the bucket %q , as it's not in the region specified by source URL", v)
+			} else if strings.Contains(err.Error(), "cannot list objects, The specified bucket does not exist") {
+				errStr = fmt.Sprintf("skip enumerating the bucket %q, as it does not exist.", v)
+			} else {
+				errStr = err.Error()
 			}
 
-			if strings.Contains(err.Error(), "cannot list objects, The specified bucket does not exist") {
-				LogStdoutAndJobLog(fmt.Sprintf("skip enumerating the bucket %q, as it does not exist.", v))
-				continue
+			// Schedule a dummy transfer so the user knows something went wrong with enumeration.
+			dummyObj := newStoredObject(
```

Comment:
It seems odd to see code like this repeated several times.  Any chance of  a reusable func newForcedErrorStoredObject()

Reply:
Yeah, that sounds like a better option considering how it's used.